src/main/java/hunternif/mc/atlas/network/PacketDispatcher.java

```diff
@@ -1,170 +1,190 @@
-			serverPackets.add(id);
-		} else {
-			clientPackets.add(id);
-			serverPackets.add(id);
-		}
-	}
-
-	/**
-	 * Send this message to everyone.
-	 */
-	public static void sendToAll(MinecraftServer server, AbstractMessage<?> message) {
-		PlayerStream.all(server).forEach(p -> ServerSidePacketRegistry.INSTANCE.sendToPlayer(p,
-				packets.inverse().get(message.getClass()),
-				toByteBuf(message)
-		));
-	}
-
-	/**
-	 * Send this message to the specified player.
-	 */
-	public static void sendTo(AbstractMessage<?> message, ServerPlayerEntity player) {
-		ServerSidePacketRegistry.INSTANCE.sendToPlayer(player,
-				packets.inverse().get(message.getClass()),
-				toByteBuf(message)
-		);
-	}
-
-	/**
-	 * Sends a message to everyone within a certain range of the coordinates in the same dimension.
-	 */
-	private static void sendToAllAround(AbstractMessage<?> message, World world, double x, double y, double z, double range) {
-		PlayerStream.around(
-				world,
-				new Vec3d(x, y, z),
-				range
-		).forEach(p -> ServerSidePacketRegistry.INSTANCE.sendToPlayer(p,
-				packets.inverse().get(message.getClass()),
-				toByteBuf(message)
-		));
-	}
-
-	/**
-	 * Sends a message to everyone within a certain range of the player provided.
-	 */
-	public static void sendToAllAround(AbstractMessage<?> message, PlayerEntity player, double range) {
-		PacketDispatcher.sendToAllAround(message, player.getEntityWorld(), player.getX(), player.getY(), player.getZ(), range);
-	}
-
-	/**
-	 * Send this message to everyone within the supplied dimension.
-	 */
-	public static void sendToDimension(AbstractMessage<?> message, World world) {
-		PlayerStream.world(world).forEach(p -> ServerSidePacketRegistry.INSTANCE.sendToPlayer(p,
-				packets.inverse().get(message.getClass()),
-				toByteBuf(message)
-		));
-	}
-
-	/**
-	 * Send this message to the server.
-	 */
-	@Environment(EnvType.CLIENT)
-	public static void sendToServer(AbstractMessage<?> message) {
-		ClientSidePacketRegistry.INSTANCE.sendToServer(
-				packets.inverse().get(message.getClass()),
-				toByteBuf(message)
-		);
-	}
+//    private static final BiMap<ResourceLocation, Class<?>> packets = HashBiMap.create();
+//    private static final BiMap<ResourceLocation, Class<?>> clientPackets = HashBiMap.create();
+//    private static final BiMap<ResourceLocation, Class<?>> serverPackets = HashBiMap.create();
+
+    private static final String PROTOCOL_VERSION = "1";
+    public static final SimpleChannel INSTANCE = NetworkRegistry.newSimpleChannel(
+            new ResourceLocation("antiqueatlas", "main"),
+            () -> PROTOCOL_VERSION,
+            PROTOCOL_VERSION::equals,
+            PROTOCOL_VERSION::equals
+    );
+
+    private static int id = 0;
+
+    /**
+     * No need to allow instantiation
+     */
+    private PacketDispatcher() {
+    }
+
+    /**
+     * Registers all packets and handlers
+     */
+//    public static void registerPackets() {

```

Comment:
TODO: delete these comments, or refactor. Search the code to see the commented-out lines where this PacketDispatcher was used previously. Looks like these methods have replacements ready.